android/src/main/kotlin/com/adyen/checkout/flutter/utils/ConfigurationMapper.kt

```diff
@@ -375,6 +376,16 @@ object ConfigurationMapper {
         }
     }
 
+    fun InstantPaymentConfigurationDTO.mapToGooglePayCheckoutConfiguration(amount: Amount): CheckoutConfiguration {
+        return CheckoutConfiguration(
+            environment.mapToEnvironment(),
+            clientKey,
+            shopperLocale?.let { Locale.forLanguageTag(it) },
```

Comment:
Does `Locale.forLanguageTag` throw an exception when the language tag is invalid?